<ui version="4.0" >
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>10</y>
     <width>141</width>
     <height>43</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:12pt; font-weight:600;">Installer &lt;/span>&lt;span style=" font-size:12pt; font-weight:600;">Settings&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
  <widget class="QGroupBox" name="optionsGroupBox" >
   <property name="geometry" >
    <rect>
     <x>350</x>
     <y>360</y>
     <width>231</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title" >
    <string>Options</string>
   </property>
   <layout class="QVBoxLayout" >
    <property name="spacing" >
     <number>6</number>
    </property>
    <property name="leftMargin" >
     <number>9</number>
    </property>
    <property name="topMargin" >
     <number>9</number>
    </property>
    <property name="rightMargin" >
     <number>9</number>
    </property>
    <property name="bottomMargin" >
     <number>9</number>
    </property>
    <item>
     <widget class="QCheckBox" name="displayTitlePage" >
      <property name="text" >
       <string>display title page on every start</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="installDetails" >
      <property name="toolTip" >
       <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">activating this option may slow down installation &lt;/p>&lt;/body>&lt;/html></string>
      </property>
      <property name="text" >
       <string>show install details</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="autoNextStep" >
      <property name="toolTip" >
       <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">select this checkbox to download, then remove, then install without any required user interaction&lt;/p>&lt;/body>&lt;/html></string>
      </property>
      <property name="text" >
       <string>auto download-remove-install </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="proxyGroupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>360</y>
     <width>331</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title" >
    <string>Proxy settings</string>
   </property>
   <widget class="QRadioButton" name="proxyOff" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>direct connection</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="proxyIE" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>83</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>IE settings</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="proxyManual" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>manual settings</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="proxyHost" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>80</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="proxyPort" >
    <property name="geometry" >
     <rect>
      <x>270</x>
      <y>80</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>170</x>
      <y>60</y>
      <width>22</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Host</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>280</x>
      <y>60</y>
      <width>46</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>Port</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="proxyFireFox" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Firefox settings</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="proxyUserName" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>120</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>100</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10" >
    <property name="geometry" >
     <rect>
      <x>240</x>
      <y>100</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Passsword</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="proxyPassword" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>120</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>390</x>
     <y>520</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="installGroupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>40</y>
     <width>571</width>
     <height>151</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;/p>&lt;/body>&lt;/html></string>
   </property>
   <property name="title" >
    <string>Install settings</string>
   </property>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>104</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">In this directory KDE applications will be installed. &lt;/p>&lt;/body>&lt;/html></string>
    </property>
    <property name="text" >
     <string>Installation Root Path</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rootPathEdit" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>30</y>
      <width>330</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="rootPathSelect" >
    <property name="geometry" >
     <rect>
      <x>480</x>
      <y>29</y>
      <width>77</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6" >
    <property name="geometry" >
     <rect>
      <x>44</x>
      <y>62</y>
      <width>70</width>
      <height>14</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">KDE on windows supports mingw and msvc compiler. Because packages containing c++ code are binary incompatible between compilers, this option let you preselect the type of compiler. If a specific compiler is selected, packages build for other compiler are invisible. &lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">Warning: Using the &lt;span style=" font-weight:600;">unspecified&lt;/span> type you should really know which packages are intended for which compiler, otherwise you may corrupt your installation. &lt;/p>&lt;/body>&lt;/html></string>
    </property>
    <property name="text" >
     <string>Compiler Type</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="compilerMinGW" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>61</y>
      <width>57</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>MinGW</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="compilerMSVC" >
    <property name="geometry" >
     <rect>
      <x>187</x>
      <y>61</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>MSVC</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="compilerUnspecified" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>244</x>
      <y>61</y>
      <width>78</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>unspecified</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="createStartMenuEntries" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>120</y>
      <width>340</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Create startmenu entries for desktop files</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="installModeEndUser" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>90</y>
      <width>67</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>End User</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="installModeDeveloper" >
    <property name="geometry" >
     <rect>
      <x>197</x>
      <y>90</y>
      <width>73</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Developer</string>
    </property>
    <property name="checked" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11" >
    <property name="geometry" >
     <rect>
      <x>56</x>
      <y>91</y>
      <width>58</width>
      <height>14</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">The install mode determines which paket components are installable. &lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">In &lt;span style=" font-weight:600;">End User mode&lt;/span> only components required for running application are installable, in &lt;span style=" font-weight:600;">Developer mode&lt;/span> development files and sources are installable too.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-weight:600;">Warning: &lt;/span>Changing the install mode after packages are already installed may result in a non usable installation&lt;/p>&lt;/body>&lt;/html></string>
    </property>
    <property name="text" >
     <string>Install Mode</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="downloadGroupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>200</y>
     <width>571</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title" >
    <string>Download settings</string>
   </property>
   <widget class="QLineEdit" name="tempPathEdit" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>23</y>
      <width>330</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="nestedDownloadTree" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>51</y>
      <width>316</width>
      <height>19</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">nested directory tree means to place every downloaded file into a separate sub directory. &lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">If checked the file dbus-1.2.3-bin.zip for example would be placed into the subdirectory dbus/dbus-1.2.3. &lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">If this option is not ckecked all downloaded files are placed directly into the package download path. &lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;/p>&lt;/body>&lt;/html></string>
    </property>
    <property name="text" >
     <string>use nested directory tree (cygwin like)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>11</x>
      <y>20</y>
      <width>114</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">In this location all downloaded files and archives are temporary saved&lt;/p>&lt;/body>&lt;/html></string>
    </property>
    <property name="text" >
     <string>Package Download Path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tempPathSelect" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>480</x>
      <y>20</y>
      <width>77</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text" >
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="mirrorGroupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>290</y>
     <width>571</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title" >
    <string>Mirror Settings</string>
   </property>
   <widget class="QComboBox" name="downloadMirror" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>20</y>
      <width>330</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="downloadMirrorLabel" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>18</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip" >
     <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">There are several download mirrors available. You can select one of them. &lt;/p>&lt;/body>&lt;/html></string>
    </property>
    <property name="text" >
     <string>Download Mirror</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="addMirrorButton" >
    <property name="geometry" >
     <rect>
      <x>480</x>
      <y>20</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Add &amp;Mirror</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>rootPathEdit</tabstop>
  <tabstop>rootPathSelect</tabstop>
  <tabstop>compilerMinGW</tabstop>
  <tabstop>compilerMSVC</tabstop>
  <tabstop>compilerUnspecified</tabstop>
  <tabstop>installModeEndUser</tabstop>
  <tabstop>installModeDeveloper</tabstop>
  <tabstop>createStartMenuEntries</tabstop>
  <tabstop>tempPathEdit</tabstop>
  <tabstop>tempPathSelect</tabstop>
  <tabstop>nestedDownloadTree</tabstop>
  <tabstop>downloadMirror</tabstop>
  <tabstop>addMirrorButton</tabstop>
  <tabstop>proxyOff</tabstop>
  <tabstop>proxyIE</tabstop>
  <tabstop>proxyFireFox</tabstop>
  <tabstop>proxyManual</tabstop>
  <tabstop>proxyHost</tabstop>
  <tabstop>proxyPort</tabstop>
  <tabstop>proxyUserName</tabstop>
  <tabstop>proxyPassword</tabstop>
  <tabstop>displayTitlePage</tabstop>
  <tabstop>installDetails</tabstop>
  <tabstop>autoNextStep</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>580</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>580</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
